.github/workflows/security.yaml

```diff
@@ -72,10 +72,19 @@ jobs:
         with:
           distribution: "goreleaser-pro"
           version: "2.3.2"
-          args: "release --clean --split --snapshot --single-target --skip=chocolatey"
+          args: "release --clean --split --snapshot --single-target"
         env:
           GORELEASER_KEY: "${{ secrets.GORELEASER_KEY }}"
       - name: "Obtain container image to scan"
         run: 'echo "IMAGE_VERSION=$(jq .version dist/linux_amd64_v1/metadata.json --raw-output)" >> $GITHUB_ENV'
-      - name: "run trivy on release image"
-        run: "docker run -v /var/run/docker.sock:/var/run/docker.sock aquasec/trivy image --format table --exit-code 1 --ignore-unfixed --vuln-type os,library --no-progress --severity CRITICAL,HIGH,MEDIUM authzed/spicedb:v${{ env.IMAGE_VERSION }}-amd64"
+      - uses: "aquasecurity/trivy-action@master"
+        name: "run trivy on release image"
+        with:
+          scan-type: "image"
```

Comment:
Use the action instead of the hand-rolled command

Reply:
I'll walk this back out with the changes to the DBs that @josephschorr put through.